Scripts/RemediationScripts/Remediate-DontUseNonADIdentities.ps1

```diff
@@ -382,33 +420,39 @@ function Remove-AzTSNonADIdentities
                 $headers = $classicAssignments.GetAuthHeader()
                 $res = $classicAssignments.DeleteClassicRoleAssignment([string] $armUri, [string] $method,[psobject] $headers)
 
-                if(($null -ne $res) -and ($res.StatusCode -eq 202 -or $res.StatusCode -eq 200))
+                    if(($null -ne $res) -and ($res.StatusCode -eq 202 -or $res.StatusCode -eq 200))
```

Comment:
Please fix the indentation of this and the enclosing block(s).

Reply:
Done.